corehq/apps/linked_domain/ucr.py

```diff
@@ -76,7 +81,8 @@ def create_linked_ucr(domain_link, report_config_id):
 
 
 def get_downstream_report(downstream_domain, upstream_report_id):
-    for linked_report in get_report_configs_for_domain(downstream_domain):
+    for linked_report in get_report_configs_for_domain(downstream_domain) + \
+            get_report_configs_for_domain(downstream_domain):
```

Comment:
I think this should be:
```suggestion
            get_registry_report_configs_for_domain(downstream_domain):
```